cvat/schema.yml

```diff
@@ -5918,6 +6510,22 @@ components:
       - name
       - related_files
       - width
+    FrameSelectionMethodEnum:
+      enum:
+      - random_uniform
+      - manual
+      type: string
+      description: |-
+        * `random_uniform` - RANDOM_UNIFORM
+        * `manual` - MANUAL
+    ImportanceEnum:
```

Comment:
FWIW, I think this type of value is usually called "severity". Consider using this term.

Reply:
Changed